integration/util/yaml.go

```diff
@@ -0,0 +1,114 @@
+package util
+
+import (
+	"context"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+
+	corev1alpha1 "github.com/giantswarm/apiextensions/v3/pkg/apis/core/v1alpha1"
+	securityv1alpha1 "github.com/giantswarm/apiextensions/v3/pkg/apis/security/v1alpha1"
+	"github.com/giantswarm/microerror"
+	corev1 "k8s.io/api/core/v1"
+	apierrors "k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/runtime"
+	capzv1alpha3 "sigs.k8s.io/cluster-api-provider-azure/api/v1alpha3"
+	expcapzv1alpha3 "sigs.k8s.io/cluster-api-provider-azure/exp/api/v1alpha3"
+	capiv1alpha3 "sigs.k8s.io/cluster-api/api/v1alpha3"
+	expcapiv1alpha3 "sigs.k8s.io/cluster-api/exp/api/v1alpha3"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+	"sigs.k8s.io/yaml"
+)
+
+func CreateCRsInFolder(ctx context.Context, client client.Client, crsFolder string) error {
+	return filepath.Walk(crsFolder, func(path string, info os.FileInfo, err error) error {
+		if !info.IsDir() {
```

Comment:
Worth checking the file's extension? here we would try apply any file.
Not a blocker, just an improvement